.github/workflows/test.yaml

```diff
@@ -19,10 +21,10 @@ jobs:
     timeout-minutes: 5
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Go 1.18
+      - name: Set up Go 1.17
```

Comment:
let's keep these for 1.18. We still want to use the latest compiler but we don't want to use features from 1.18

https://go.dev/ref/mod#go-mod-file-go